src/poliastro/tests/test_maneuver.py

```diff
@@ -61,6 +61,31 @@ def test_hohmann_maneuver():
     )
 
 
+def test_hohmann_maneuver_not_at_pericenter():
+
```

Comment:
Instead of creating a new test, I propose parametrizing `test_hohmann_maneuver`, which is already using the data from example 6-1, and instead of starting from a circular orbit start from an elliptical one, whose pericenter matches the expected altitude. Therefore, when the anomaly is 0, we are in the case where the time to pericenter will be zero, and if we move to a different starting anomaly, we will have a different time.